src/Symfony/Bundle/WebProfilerBundle/DependencyInjection/WebProfilerExtension.php

```diff
@@ -53,6 +56,11 @@ public function load(array $configs, ContainerBuilder $container)
             $container->setParameter('web_profiler.debug_toolbar.intercept_redirects', $config['intercept_redirects']);
             $container->setParameter('web_profiler.debug_toolbar.mode', $config['toolbar'] ? WebDebugToolbarListener::ENABLED : WebDebugToolbarListener::DISABLED);
         }
+
+        if (Kernel::VERSION_ID >= 30408 || Kernel::VERSION_ID >= 40008) {
```

Comment:
this needs to be `Kernel::VERSION_ID >= 30408 && Kernel::VERSION_ID < 40000 || Kernel::VERSION_ID >= 40008)`

Reply:
fixed in 946eefa